.travis.yml

```diff
@@ -3,9 +3,11 @@ rvm:
   - 2.1.9
   - 2.2.5
   - 2.3.1
+  - 2.4.0
   - jruby-9.0.5.0
 
 gemfile:
   - Gemfile
+  - ./turnip2.gemfile
```

Comment:
It calls bundler with the file. However for some reason it needs a directory in the file name